src/script/CMakeLists.txt

```diff
@@ -1,8 +1,11 @@
 # Aseprite Scripting Library
 # Copyright (C) 2015-2016  David Capello
+# Copyright (C) 2021  LibreSprite contributors
```

Comment:
I guess it'd make sense to rename it in LibreSprite Scripting Library, as it's now being modified by us